.github/workflows/publish-docs.yml

```
@@ -30,31 +30,33 @@ jobs:
-        uses: actions/setup-python@0b93645e9fea7318ecaed2b359559ac225c90a2b # v5.3.0
+      - name: 🐍 Install uv and set Python ${{ matrix.python-version }}
+        uses: astral-sh/setup-uv@0c5e2b8115b80b4c7c5ddf6ffdd634974642d182 # v5.4.1
         with:
           python-version: ${{ matrix.python-version }}
 
-      - name: 📜 Setup Poetry
-        uses: abatilo/actions-poetry@e78f54a89cb052fff327414dd9ff010b5d2b4dbd # v3.0.1
 
-      - name: 📦 Install dependencies
+      - name: 🏗️ Install dependencies
         run: |
```

actionlint (1.7.4)

property "mkdocs_token" is not defined in object type {}

(expression)